app/renderer/actions/Session.js

```diff
@@ -724,6 +727,54 @@ export function setSavedServerParams () {
   };
 }
 
+export function setStateFromAppiumFile (newFilepath = null) {
+  return (dispatch) => {
+    // no "fs" means not a browser. do nothing
+    if (!fs) {
```

Comment:
when would this function ever be called in a non-browser context? it's in the renderer after all

Reply:
I worded that incorrectly. The comment should say `no "fs" means it's not an electron renderer` to indicate that it's being run from a browser.